src/sprout/Controllers/DbToolsController.php

```diff
@@ -2281,7 +2280,8 @@ public function moduleBuilderExistingUploadAction()
             if (!preg_match('!^[a-zA-Z0-9]+$!', $_POST['existing'])) {
```

Comment:
we could probably replace this funky validation here with a `$module === null`.

Reply:
I pushed something for that